sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -4,28 +4,35 @@ Azure Time Series Insights provides data exploration and telemetry tools to help
 
 Azure Time Series Insights gives you a global view of your data, so you can quickly validate your IoT solution and avoid costly downtime to mission-critical devices. It can help you discover hidden trends, spot anomalies, and conduct root-cause analysis in near real time.
 
-If you are new to Azure Time Series Insights and would like to learn more about the platform, please make sure you check out the Azure Time Series Insights official documentation page.
+If you are new to Azure Time Series Insights and would like to learn more about the platform, please make sure you check out the Azure Time Series Insights official [documentation page][tsi_product_documentation].
 
-# Time Series Insights Samples
+# Time Series Insights client library for .NET
+The Time Series Insights client library for .NET provides the following functionality:
+- Retrieving and being able to make changes to the Time Series Insights environment model settings, such as changing the model name or default type ID.
+- Retrieving and being able to add, update and remove Time Series instances.
+- Retrieving and being able to make changes to the Time Series Insights environment types, such as creating, updating and deleting Time Series types.
+- Retrieving and being able to make changes to the Time Series Insights hierarchies, such as creating, updating and deleting Time Series hierarchies.
+- Executing Time series queries such as querying for raw events, series and aggregate series
+
+[Source Code][tsi_client_src] | [Package (NuGet)][tsi_nuget_package] | [Product documentation][tsi_product_documentation] | [Samples][tsi_samples]
+
+# Key concepts
+
+## TimeSeriesInsightsClient
 
-You can explore the time series insights APIs (using the client library) using the samples project.
+A `TimeSeriesInsightsClient` is the primary interface for developers using the Time Series Insights client library. It provides both synchronous and asynchronous operations to perform operations on a Time Series Insights environment. The `TimeSeriesInsightsClient` exposes several properties that a developer will use to perform specific operations on a Time Series Insights environment. For example, `ModelSettings` is the the property that a developer can use to perform operations on the model settings of the TSI environment. `Instances` can be used to perform operations on TSI instances. Other properties include `Types`, `Hierarchies` and `Query`.
 
-The samples project demonstrates the following:
+# Creating TimeSeriesInsightsClient
 
-- Instantiate the client
-- Demonstrate how to use Model APIs
-    - Create, get, replace, and delete instances
-    - Create, get, replace, and delete types
-    - Create, get, replace, and delete hierarchies
-    - Get and update model configuration settings
-- Demonstrate how to query raw events, series, and aggregates.
+To create a new Time Series Insights client, you need the endpoint to an Azure Time Series Insights instance and supply credentials.
```

Comment:
I know instance here is generic (not the model instance) but it could be confusing. Can we change it or remove?

Reply:
how about change instance to environment?

Reply:
yeah that works